extract.js

```diff
@@ -169,8 +169,9 @@ class Extract extends Writable {
 
     this._stream = this._createStream()
     this._missing = this._header.size
+    const header = { ...this._header, contentsByteOffset: this._buffer.shifted }
```

Comment:
i think we can name it byteOffset just, similar to how that property is normally used.

in headers.decode, just declare it there and init it to 0, then simply mutate it in here instead of the spread

Reply:
I've applied these changes. Thank you! 🙏